common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -265,45 +262,48 @@ public class CommonValue {
                         + MLModel.LAST_UNDEPLOYED_TIME_FIELD
                         + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
                         + "      \""
+                        + MLModel.INTERFACE_FIELD
+                        + "\": {\"type\": \"flat_object\"},\n"
+                        + "      \""
```

Comment:
I see that allowedInterfaceFieldKeys only includes "input", "output". Will new allowed keys possible added in the future?  If not, why not just define a Class rather than a Map<String, String> modelInterface? Using the Map is primarily for extensibility.